File: FilmPoolWeb/src/app/models/film.model.ts
import { Genre } from "../enums/genre.enum";

 
export class Film {
    id!: number;
    title!: string;
    genre!: Genre;
    duration!: string;
    year!: number;
    description!: string;
    picture!: string;
    rating!: number;
    filmUrl!: string;

}

File: FilmPool/ResponseModels/FilmModel.cs
using FilmPool.DbModels;
using System.Drawing;

namespace FilmPool.ResponseModels
{
  public class FilmModel
  {
    public int Id { get; set; }
    public string Title { get; set; }
    public GenreEnum Genre { get; set; }
    public Genre FilmGenre { get; set; }
    public int? Year { get; set; }
    public string Duration { get; set; }
    public string? Description { get; set; }
    public string? Picture { get; set; }
    public double? Rating { get; set; }

  }
}
